### ISSUE TITLE
How do you use this on actual books?

### ISSUE THREAD

**Issue by Baniita** [ritiek/scribd-downloader#15]
Hello, is this only for documents, and not books? So it's not going to work for this, right? https://www.scribd.com/read/189087235

Erik's version doesn't work for books either. I figured since it can text scrape, that it'd be able to but... :( 

All I get is a message saying it's extracting the text, but the .txt file is nowhere to be found. I'm assuming the working folder is the cd path I'm on atm? ...Yeah, nowhere

**Comment by ritiek** [comment]
Nope, it doesn't work books at the moment but I feel it should be possible to extract text from books at least in this script. I'll go take a look.

**Comment by ritiek** [comment]
v0.1.6 can now download preview version of books to PDF. Upgrade scribd-downloader with:
```
pip install scribd-downloader -U
```
and download the book with
```
scribdl https://www.scribd.com/read/189087235/Confessions-of-a-Casting-Director-Help-Actors-Land-Any-Role-with-Secrets-from-Inside-the-Audition-Room
```

I couldn't find any way to download the full book. Scribd doesn't seem to expose the contents of the full book and just exposes the preview version (unlike documents where it exposes the full document content). 

If anyone has a Scribd premium account and would like to share it with me, please e-mail me. I might be able to reverse engineer it and allow downloading of the full book with this tool.

**Comment by ritiek** [comment]
Also see #28.

**Comment by ritiek** [comment]
Closed by #32. You can now download complete books.
